### DISCUSSION TITLE
Code coverage in Storybook 8

### DISCUSSION THREAD

**Discussion by xgebi** [storybookjs/storybook#26877] [Help]
### Summary

So, I'm trying to upgrade from v7.6.4 to v8.0.8. Quite a few tests started failing after the upgrade with

```
[Test runner] An error occurred when evaluating code coverage:
      The code in this story is not instrumented, which means the coverage setup is likely not correct.
      More info: https://github.com/storybookjs/test-runner#setting-up-code-coverage
```

When I checked with [docs](https://github.com/storybookjs/test-runner#setting-up-code-coverage) it was still referring to jest. Is that information still correct?

As for the story, that fails it looks like this:

``` TypeScript
export const Default: Story = {
    render: Template,
};
```

There's no `play` function. In this format pure CSS components pass but not components based on lit.

**Comment by umema4** [comment]
same here.
"storybook": "^8.1.9",
"@storybook/test-runner": "^0.18.2",
"@storybook/addon-coverage": "^1.0.4",

**Comment by cglango** [comment]
Anyone find a solution?
